scripts/log.ps1

```diff
@@ -196,11 +196,15 @@ function Log-Stop {
             Write-Debug "Decoding LTTng into BabelTrace format ($BableTraceFile)"
             babeltrace --names all $TempDir/* > $BableTraceFile
 
-            Write-Host "Decoding into human-readable text: $ClogOutputDecodeFile"
-            $Command = "$Clog2Text_lttng -i $BableTraceFile -s $SideCar -o $ClogOutputDecodeFile --showTimestamp --showCpuInfo"
-            Write-Host $Command
-            Invoke-Expression $Command | Write-Debug
-            Remove-Item -Path $BableTraceFile -Force | Out-Null
+            Write-Host "Babeltrace ran. Compile clog2text and run the following command"
+            $Command = "clog2text_lttng -i $BableTraceFile -s $SideCar -o $ClogOutputDecodeFile --showTimestamp --showCpuInfo"
+            $Command
+
+            # Write-Host "Decoding into human-readable text: $ClogOutputDecodeFile"
+            # $Command = "$Clog2Text_lttng -i $BableTraceFile -s $SideCar -o $ClogOutputDecodeFile --showTimestamp --showCpuInfo"
+            # Write-Host $Command
+            # Invoke-Expression $Command | Write-Debug
+            # Remove-Item -Path $BableTraceFile -Force | Out-Null
```

Comment:
Undo this?

Reply:
Nope. Clog2text_lttng was broken by the clog changes a few months ago. Theres currently no setup to build and install clog2text_lttng on the test machines. I need to file another bug to fix that, but its a bigger task.